tests/unit/systems/dag/test_executors.py

```diff
@@ -26,6 +26,7 @@
 from merlin.schema import ColumnSchema, Schema
 from merlin.systems.dag.ensemble import Ensemble
 from merlin.systems.dag.ops.session_filter import FilterCandidates
+from merlin.systems.dag.runtime.triton import TritonExecutorRuntime
```

Comment:
Doesn't seem like these tests belong in `test_executors.py`, since what's really being tested isn't the executors but the runtimes. Maybe worth a new `test_runtimes.py` file?